build.gradle

```diff
@@ -44,6 +44,10 @@ subprojects {
   configurations.all {
     resolutionStrategy.cacheChangingModulesFor 0, "minutes"
   }
+
+  tasks.withType(JavaCompile).configureEach {
+    options.compilerArgs << '-parameters'
+  }
```

Comment:
what does this do?

Reply:
This adds the `-parameters` flag to the compilation so that we preserve parameter names, which are now required for some reflective operations because of a class that was removed in Spring 6.1. This flag was mentioned in the 6.1 change notes.